console-consumer-primitive-keys-values/README.md

```diff
@@ -0,0 +1,61 @@
+# Consuming non-string primitive keys and values 
+
+You want to inspect or debug records written to a topic. Each record key and value is a long and double, respectively. In this tutorial, you'll learn how to specify key and value deserializers with the console consumer.
```

Comment:
Since this change takes the tutorials from AK CLIs + `confluent` CLI to just the AK CLIs, suggest making it clear in both the title and intro that this is just the AK tools, e.g. "how to specify key and value deserializers with the Apache Kafka console consumer." This applies to all 4 tutorials